### PR TITLE
Add AGM news article

### PR DESCRIPTION
**Pull request by AaronC81**
Identical to the email, besides plugging the mailing list when email is first mentioned

### CODE REVIEW

**Comment by sersorrel** [comment]
not everything is wrapped to 70 characters as the README says it should be, but lgtm

**Comment by AaronC81** [comment]
Cheers, rewrapped it now!

**Review by adambirtles — DISMISSED** [review]

File: news/2021-02-26-agm.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+**Friday 5th March at 6pm**!
+
+This year's AGM will be held online. If you have nominated yourself
+for a committee position, join using [this Google Meet
+link](https://meet.google.com/oqe-djht-bmo) so that you can speak.
+
+If you aren't thinking of nominating yourself for a position, then
+please watch the live stream of the meeting instead! This will be
+streamed using [our live streaming service](https://live.hacksoc.org/)
+- a direct link to the stream will be emailed to you when the AGM
```
Comment (adambirtles) [inline news/2021-02-26-agm.md:15]: Oops, almost missed this. Seems like the line wrapped on a hyphen here and created a rogue list.

Comment (AaronC81) [reply news/2021-02-26-agm.md]: Good catch, should be fixed now :)

**Review by adambirtles — APPROVED** [review]

**Pull request merged**